src/features/offline-checks/model/services/MediaLookupService.ts

```diff
@@ -0,0 +1,87 @@
+import { LookupMediaInput } from '@app/abstract/lib';
+import { ActivityDetails, ActivityModel } from '@app/entities/activity';
+import { ActivityResponse, AppletDetailsResponse } from '@app/shared/api';
+import {
+  getActivityDetailsKey,
+  getAppletDetailsKey,
+  getDataFromQuery,
+} from '@app/shared/lib';
+
+const createMediaLookupService = () => {
+  const lookupInMarkdown = (message: string): boolean => {
```

Comment:
Maybe it would be better to name the functions with the "includes" in it.
Like, lookupInMarkdown => markdownIncludesMedia
lookupInActivity => activityIncludesMedia